<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PowerChatClient</class>
 <widget class="QMainWindow" name="PowerChatClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PowerChatClient</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QLabel" name="label_userName">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="label_status">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_activities">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>65</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Cascadia Code</family>
       </font>
      </property>
      <property name="title">
       <string>Activities</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_icon">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(:/icon.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_card">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:hover{
  background-color:rgb(85, 170, 255);
  color:rgb(0,0,0);
}</string>
         </property>
         <property name="text">
          <string>打卡</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_addFriends">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>＋</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_deleteFriends">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>6</pointsize>
          </font>
         </property>
         <property name="text">
          <string>➖</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_status">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <item>
          <property name="text">
           <string>离线</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>在线</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Q我吧</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>离开</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>忙碌</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>请勿打扰</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>隐身</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_contactSwitch">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>65</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Cascadia Code</family>
       </font>
      </property>
      <property name="title">
       <string>Contact Switch</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="btn_friends">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>好友</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_chatRoom">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>群聊</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/head_chenglong.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="titleBarGroup">
      <item>
       <widget class="QPushButton" name="btn_min">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>.萍方-简</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
   color:white;
}
QPushButton:hover{
  background-color:rgb(85, 170, 255);
  color:rgb(0,0,0);
}</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_close">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>.萍方-简</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
   color:white;
}
QPushButton:hover{
  background-color:rgb(212, 64, 39);
  color:rgb(0,0,0);
}</string>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox_contactList">
      <property name="font">
       <font>
        <family>Cascadia Code</family>
       </font>
      </property>
      <property name="title">
       <string>Contact List</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="0" colspan="3">
        <widget class="QLabel" name="label_log_title">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>PowerChat 消息：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_offline_hint">
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>离线：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QPushButton" name="btn_contact_disconnect">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>不想和你聊了 😕</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_online_hint">
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>在线：</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <widget class="QPushButton" name="btn_iknow">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>分かりました 💕</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QPushButton" name="btn_contact_open">
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
         <property name="text">
          <string>我们一起聊天吧 😀</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QTextBrowser" name="textBrowser">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox_online">
         <property name="minimumSize">
          <size>
           <width>190</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_online_num">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox_offline">
         <property name="minimumSize">
          <size>
           <width>190</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_offline_num">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>btn_card</tabstop>
  <tabstop>btn_addFriends</tabstop>
  <tabstop>comboBox_status</tabstop>
  <tabstop>btn_friends</tabstop>
  <tabstop>btn_chatRoom</tabstop>
  <tabstop>comboBox_online</tabstop>
  <tabstop>comboBox_offline</tabstop>
  <tabstop>btn_contact_disconnect</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
